tests/test_timeable_mixin.py

```
@@ -24,69 +23,84 @@
+    np.testing.assert_almost_equal(duration, 2, decimal=1)
+
+    T.decorated_takes_time_auto_key(num_seconds=2)
+    duration = T._times_for("decorated_takes_time_auto_key")[-1]
+    np.testing.assert_almost_equal(duration, 2, decimal=1)
+
+    T.decorated_takes_time(num_seconds=1)
+    stats = T._duration_stats
+
+    assert {"decorated", "decorated_takes_time_auto_key"} == set(stats.keys())
+    np.testing.assert_almost_equal(1.5, stats["decorated"][0], decimal=1)
+    assert 2 == stats["decorated"][1]
```

Ruff

Yoda condition detected

Rewrite as `stats["decorated"][1] == 2`

(SIM300)